roles/slave_tools/tasks/redis.yml

```diff
@@ -7,4 +7,13 @@
   - redis-server
   - redis-tools
   tags: redis
+- name: Do not listen on IPv6
+  lineinfile:
+    dest: /etc/redis/redis.conf
+    regexp: '::1'
+    state: absent
+- name: Listen on IPv4 loopback
+  lineinfile:
+    dest: /etc/redis/redis.conf
+    line: 'bind 127.0.0.1'
```

Comment:
Why?

Reply:
It was the default before. In newer version, it also tries to bind on the IPv6 localhost, which is not available in containers, so we force the IPv4-only bind.